docs/reference/rest-api/security/role-mapping-resources.asciidoc

```diff
@@ -70,7 +70,11 @@ example, `"username": "jsmith"`.
 `groups`::
 (array of strings) The groups to which the user belongs. For example, `"groups" : [ "cn=admin,ou=groups,dc=example,dc=com","cn=esusers,ou=groups,dc=example,dc=com ]`.
 `metadata`::
-(object) Additional metadata for the user. This can include a variety of key-value pairs. When referencing metadata fields in role mapping rules, use the dot notation to specify the key within the metadata object. If the key contains special characters such as parentheses, dots, or spaces, you must escape these characters using backslashes (`\`). For example, `"metadata": { "cn": "John Smith" }`.
+
+(object)  Additional metadata for the user is passed as key-value pairs. For example, `"metadata": { "cn": "John Smith" }`.
+
+When referencing metadata fields in role mapping rules, use dot notation to specify the key within the metadata object. If the key contains special characters such as parentheses, dots, or spaces, you must escape these characters using backslashes (`\`). 
```

Comment:
I think we can add a `+` to get this formatted so it's indented (it's clearer the content is all related if it's indented)

```suggestion
+
When referencing metadata fields in role mapping rules, use dot notation to specify the key within the metadata object. If the key contains special characters such as parentheses, dots, or spaces, you must escape these characters using backslashes (`\`). 
```